src/chrome/content/rules/Pingupod.de.xml

```diff
@@ -3,7 +3,6 @@
 	<!--	Direct rewrites:
 				-->
 	<target host="pingupod.de" />
-	<target host="www.pingupod.de" />
```

Comment:
Use `<target host=" " />` instead of test urls to cover a whole host.
`<rule from="^http://(www|mgp|diaspora)\.pingupod\.de/"
    to="https://pingupod.de/" />` is not necessary because `<rule from="^http:"
 		to="https:" />` redirects all targets.

With this simplified syntax test urls are not required.